templates/partials/modal_reply.haml

```diff
@@ -3,53 +3,55 @@
 .modal-header.no-border
   %h3.modal-title
     - trans "Reply"
-  .modal-body.no-bottom-padding{ ng-init:'init()' }
```

Comment:
I've noticed this on all the other modals as well, it works/looks much better when not nested